mitosheet/src/components/endo/ColumnHeader.tsx

```diff
@@ -361,7 +342,7 @@ const ColumnHeader = (props: {
                             }}
                             // We submit the column header if the user focuses outside the input
                             onBlur={() => {
-                                void submitRenameColumnHeader();
+                                void _submitRenameColumnHeader();
```

Comment:
I think you can call submit function directly here, as you never need to pass an e value. Wrapping it might just obfuscate things...

Reply:
It's a bit more verbose, but also more clear I think!